mate-base/mate-desktop/mate-desktop-1.26.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+fi
+
+DESCRIPTION="Libraries for the MATE desktop that are not part of the UI"
+LICENSE="FDL-1.1 GPL-2+ LGPL-2+ MIT-with-advertising"
+SLOT="0"
+
+IUSE="X debug +introspection startup-notification"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.50:2
+	>=gnome-base/dconf-0.13.4
+	x11-libs/cairo
+	x11-libs/libX11
+	>=x11-libs/libXrandr-1.3
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	introspection? ( >=dev-libs/gobject-introspection-0.9.7:= )
+	startup-notification? ( >=x11-libs/startup-notification-0.5:0 )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
```

Comment:
It's unlikely this belongs in only RDEPEND. It should be in COMMON_DEPEND. I don't think libintl is ever dlopen()'d.

Reply:
Done.